common/common-api/src/main/java/org/ow2/proactive/permissions/RoleUtils.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.permissions;
+
+import java.lang.reflect.Method;
+
+import org.apache.log4j.Logger;
+
+
+public class RoleUtils {
+
+    static final String ROLE_BASIC = "basic";
+
+    static final String ROLE_READ = "read";
+
+    static final String ROLE_WRITE = "write";
+
+    static final String ROLE_PROVIDER = "provider";
+
+    static final String ROLE_NSADMIN = "nsadmin";
+
+    static final String ROLE_ADMIN = "admin";
+
+    static final String ROLE_NONE = "none";
```

Comment:
@mklkun I could yes, though only the string value matters

Reply:
For me it is better to have an enum in case we will need these Roles in other places in the future and for a better code readability. But it is not that crucial. It is ok for me.

Reply:
I'm on it